### PR TITLE
Cache the font files

### PR DESCRIPTION
**Pull request by shamwela**
I cached the font files for a year to improve performance.

### CODE REVIEW

File: beta/vercel.json [hunk 1]
```diff
@@ -254,5 +254,43 @@
       "destination": "https://5a046bf5a6188f4b8fa4938a--reactjs.netlify.com",
       "permanent": false
     }
+  ],
+  "headers": [
+    {
+      "source": "/fonts/SourceCodePro-Regular.ttf",
```
Comment (harish-sethuraman) [inline beta/vercel.json:260]: There should probably be a way to add all font files inside the `fonts` like `/fonts/(.*).ttf` ?

Comment (shamwela) [reply beta/vercel.json]: Thanks. I've improved the syntax.

File: beta/vercel.json [hunk 2]
```diff
@@ -254,5 +254,16 @@
       "destination": "https://5a046bf5a6188f4b8fa4938a--reactjs.netlify.com",
       "permanent": false
     }
+  ],
+  "headers": [
+    {
+      "source": "/fonts(.*)",
```
Comment (harish-sethuraman) [inline beta/vercel.json:260]:
```suggestion
      "source": "/fonts/.*",
```
Also we might want to check if this works or we need to do `\/fonts\/.*` cc: @gaearon

**Comment by gaearon** [comment]
I don’t think we want to hold them in the same repo since they’re not provided by the same license.

**Comment by shamwela** [comment]
I didn't know about the licenses. Should I only cache the "Source Code Pro Regular" font file?

**Comment by gaearon** [comment]
Let me chat to some folks on our side and then report back

**Comment by shamwela** [comment]
Any update on this, please?

**Comment by gaearon** [comment]
Let's do this just for SourceCode for now?

**Comment by shamwela** [comment]
I'll do that in a new pull request. Thanks.

**Pull request closed without merging**